client/src/pages/404/Page404.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import "./page404.css";
+
+type Page404Props = {
+    message: string;
+};
+
+export const Page404 = (props: Page404Props): JSX.Element => {
+    return (
```

Comment:
Можно сразу деструктурировать пропсы и обойтись без фигурных скобок

```suggestion
export const Page404 = ({message}: Page404Props): JSX.Element => (
```